src/components/PatchActionButtons/addNotification/PatchNotificationModal.tsx

```diff
@@ -0,0 +1,157 @@
+import React from "react";
+import { NotificationModal } from "components/NotificationModal";
+import {
+  SubscriptionMethods,
+  Trigger,
+  RegexSelector,
+} from "hooks/useNotificationModal";
+import {
+  SUBSCRIPTION_JIRA_COMMENT,
+  SUBSCRIPTION_SLACK,
+  SUBSCRIPTION_EMAIL,
+} from "types/subscription";
+import {
+  validateDuration,
+  validatePercentage,
+  validateEmail,
+  validateJira,
+  validateSlack,
+} from "utils/validators";
+import { useParams } from "react-router-dom";
```

Comment:
nit. can you Move this to the top just so external package imports come before internal ones